### DISCUSSION TITLE
Not really an issue but very helpfull

### DISCUSSION THREAD

**Discussion by antonyf** [vimeo/player.js#880] [Show and tell]
Creating a new player or multiple players

const players = new Array();

const id = randID();

function makenewplayer(t, id){
players[id] = new Vimeo.Player(t);
}

<iframe id="'+id +'" onload="makenewplayer(this, '+id+');" src=" url for video with all the extra do's and dont's "></iframe>

access the player like this:

players[id]..pause(); 
and so on..

hope this is usefull

**Comment by eballeste** [comment]
would be better to [add this snippet to the discussions board](https://github.com/vimeo/player.js/discussions) instead, while changing the title to something more descriptive, (for example, why is this helpful?) so it's easier to find
